src/index.ts

```diff
@@ -205,12 +206,13 @@ app.get('/api/moonpay-sign', async (_req, res) => {
 });
 
 app.post('/api/alice-bob/create-order', async (_req, res) => {
-  const { amount, from, to, userId, walletAddress, cardNumber } = _req.query;
+  const { isWithdraw, amount, from, to, userId, walletAddress, cardNumber } = _req.query;
+  const booleanIsWithdraw = isWithdraw === 'true';
 
   try {
     const exchangeInfo = {
-      from: String(from),
-      to: String(to),
+      from: isDefined(isWithdraw) ? (booleanIsWithdraw ? 'TEZ' : 'CARDUAH') : String(from),
+      to: isDefined(isWithdraw) ? (booleanIsWithdraw ? 'CARDUAH' : 'TEZ') : String(to),
```

Comment:
Sorry, could you move this logic into a separate function and use it here and https://github.com/madfish-solutions/templewallet-backend/pull/118/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R289 ?